crates/swc_common/src/syntax_pos/hygiene.rs

```diff
@@ -57,6 +57,10 @@ struct SyntaxContextData {
 
 /// A mark is a unique id associated with a macro expansion.
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
+#[cfg_attr(
+    feature = "rkyv",
+    derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize)
+)]
```

Comment:
If I recall correctly I beieve I intentionally did not enable serialization for this since its inner value (u32) can be exchanged between host - guest without any serialization.

Reply:
ref: https://github.com/swc-project/swc/blob/34a0fac442bacad5ef0fabbccdaf2bfabb647da4/crates/swc_common/src/syntax_pos/hygiene.rs#L79-L81

Reply:
OK, I'll revert it